src/components/Map/Controls/index.tsx

```diff
@@ -172,6 +194,18 @@ export const MapControls = () => {
                   );
                 }}
               />
+              <DisposableLinkButtonComponent
+                href="https://oytutanak.com"
+                dispose={votingLocationsFilter.isOpen}
+                buttonLabel={t("links.sendDocumentTitle")}
+                icon={<SummarizeIcon />}
+              />
+              <DisposableLinkButtonComponent
+                href="https://t.me/oytutanakbot"
```

Comment:
please make external site redirects in new tab

Reply:
they already are in new tab, disposable link component has target=_blank inside